substratest/assets.py

```diff
@@ -167,7 +167,8 @@ def load(cls, d):
 
             if hasattr(attr_type, '__origin__') and attr_type.__origin__ is typing.Union \
                     and len(attr_type.__args__) == 2 and type(None) in attr_type.__args__:
-                attr_type = [arg for arg in attr_type.__args__ if arg is not type(None)][0]
+                attr_type = [arg for arg in attr_type.__args__ if arg
+                             is not type(None)][0]  # noqa: E721
```

Comment:
The line break seems unusual. A better line break to improve readability would be before the `if`.

Reply:
Yes, you're right. I wasn't sure about this